dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/statelisteners/ElasticListener.java

```diff
@@ -349,6 +351,7 @@ public static JsonNode dockstoreEntryToElasticSearchObject(final Entry entry) th
         objectNode.set("all_authors", MAPPER.valueToTree(allAuthors));
         objectNode.set("categories", MAPPER.valueToTree(convertCategories(entry.getCategories())));
         objectNode.put("archived", entry.isArchived());
+        objectNode.put("selected_concept_doi", MAPPER.valueToTree(selectedConceptDoi));
```

Comment:
This works, but it seems like the current intended use of the various `ObjectNode.put`s is to set a field to a particular non-`JsonNode` value.  To that end, the above variety of `put`, which sets a field to a `JsonNode` value, appears to be deprecated:
https://www.javadoc.io/static/com.fasterxml.jackson.core/jackson-databind/2.18.2/com/fasterxml/jackson/databind/node/ObjectNode.html#put-java.lang.String-com.fasterxml.jackson.databind.JsonNode-
Probably better to use a `set` here.